Mathlib/Testing/SlimCheck/Testable.lean

```diff
@@ -512,11 +512,11 @@ partial def addDecorations (e : Expr) : MetaM Expr :=
     if not (← Meta.inferType e).isProp then
       return .continue
     else if let Expr.forallE name type body data := expr then
-      let n := name.toString
       let newType ← addDecorations type
-      let newBody ← addDecorations body
+      let newBody ← Meta.withLocalDecl name data type fun fvar => do
+        return (← addDecorations (body.instantiate1 fvar)).abstract #[fvar]
```

Comment:
I'm surprised that there is no `abstract1` here; I'm also surprised that there don't seem to be any other users of `abstract` in mathlib.

Reply:
normally the abstract is folded together with the construction of the forall using `mkForall`

Reply:
Are you advocating a change here?

Reply:
The documentation for `mkForall` is
```
/--
`.forallE x t b bi` is now the preferred form.
-/
def mkForall (x : Name) (bi : BinderInfo) (t : Expr) (b : Expr) : Expr :=
  .forallE x t b bi
```
which I assume should be read as a deprecation?

Reply:
sorry, I meant `mkForallFVars`

Reply:
Are you happy with this as is, or do you want me to try to rewrite to use `mkForallFVars`?